components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -1212,5 +1212,12 @@ void addFileToProductDocumentation(APIProductIdentifier prodcutId, Documentation
      */
     public void addProductDocumentationContent(APIProduct apiProduct, String documentationName, String text) throws APIManagementException;
 
-
+    /**
+     * This method used to save the graphql schema content
```

Comment:
typo : "This method **is** used to"

Reply:
Fixed in c4db271137c83d7f2a2a4c40a8c6248453a83817